newIDE/app/src/MainFrame/Changelog/ChangelogDialog.js

```diff
@@ -35,12 +40,19 @@ const ChangelogDialog = ({ open, onClose }: Props) => {
       actions={actions}
       open={open}
       onRequestClose={onClose}
+      flexColumnBody
     >
-      <Text>
-        <Trans>
-          GDevelop was upgraded to a new version! Check out the changes.
-        </Trans>
-      </Text>
+      <Line noMargin justifyContent="space-between">
```

Comment:
Should this be a ResonsiveLineStackLayout so that the button goes below on small screens?

Reply:
Good spot!